src/workers/python/init.py.ts

```diff
@@ -11,6 +11,9 @@ import html
 import types
 import os
 import micropip
+# temporarily until friendly_traceback does this automatically
+await micropip.install("stack_data")
```

Comment:
`friendly_traceback` depends on `stack_data` (which is why you need to clear the stack_data cache) so you only need to install friendly_traceback, and import stack_data after that.

Reply:
Since Safari doesn't handle those installs very well, this might be useful (I am considering giving other dependencies of friendly_traceback a similar treatment).

Reply:
Well it can be removed now anyway